frontend/src/App.tsx

```diff
@@ -34,13 +35,14 @@ const client_id =
 /**
  * The OIDC Configuration needed for authentication.
  */
-const oidcConfig = {
+const oidcConfig: AuthProviderProps = {
   authority: authority,
   client_id: client_id,
   redirect_uri: window.location.origin + "/",
   scope: "openid",
   automaticSilentRenew: true,
   revokeTokensOnSignout: true,
+  userStore: new WebStorageStateStore({ store: sessionStorage }),
```

Comment:
Added this change so that if our requirements for login ever change from one session per tab to one session per window, its a simple matter of reconfiguring to `localStorage` here.